pkg/app/app.go

```diff
@@ -102,7 +102,7 @@ func defaultFuncApiUrl(_ configuration.Configuration, logger *zerolog.Logger) co
 
 		// Same logic for PAT - if a PAT is provided, and it has a URL in the claims, use that instead
 		if auth.IsAuthTypePAT(authToken) {
-			apiUrl, claimsErr := auth.GetApiUrlFromPAT(authToken)
+			apiUrl, claimsErr := auth.GetApiUrlFromPAT(authToken, config.GetString(auth.CONFIG_KEY_ALLOWED_HOST_REGEXP))
```

Comment:
Question: is this something we should do for the apiUrl from the Oauth token as well?